python/pyarrow/ipc.py

```diff
@@ -39,10 +39,17 @@ class RecordBatchStreamReader(lib._RecordBatchStreamReader):
     ----------
     source : bytes/buffer-like, pyarrow.NativeFile, or file-like Python object
         Either an in-memory buffer, or a readable file object.
+    options : pyarrow.ipc.IpcReadOptions
+        Options for IPC serialization.
+
+        If None, default values will be used.
```

Comment:
No need for the empty line IMHO.
```suggestion
        Options for IPC deserialization.
        If None, default values will be used.
```